assets/scss/base/_lists.scss

```diff
@@ -58,11 +48,81 @@ ul, nav ul {
 }
 
 .list--inline {
-	li {
-		display: inline;
+  li {
+    display: inline;
+  }
+}
+
+/*
+Numbered List
+
+Defining sections in markup with numbers associated against them using `.list--numbered` on the container and `.numbered` on the element to identify its position
+
+Markup:
+<fieldset class="list--numbered">
+  <legend>Section 1</legend>
+  <div class="form-field-group numbered">
```

Comment:
I have only seen `.numbered` with `.form-field` the errors work for this went up earlier http://hmrc.github.io/assets-frontend/section-errors.html